react/Installments.tsx

```diff
@@ -45,22 +56,33 @@ function Installments({
     return null
   }
 
-  let [maxInstallmentOption] = commercialOffer.Installments
+  let [installmentsOption] = commercialOffer.Installments
+
+  switch (installmentsCriteria) {
+    case 'max-quantity-without-interest': {
+      installmentsOption = pickMaxInstallmentsOptionWithNoInterest(
```

Comment:
What do you think about changing it since we changed the 'max-quantity-without-interest'?
```suggestion
      installmentsOption = pickMaxInstallmentsOptionWithoutInterest(
```